torch/nn/modules/module.py

```diff
@@ -763,11 +763,9 @@ def _load_from_state_dict(self, state_dict, prefix, local_metadata, strict,
                                       .format(key, input_param.shape, param.shape))
                     continue
 
-                if isinstance(input_param, Parameter):
-                    # backwards compatibility for serialized parameters
-                    input_param = input_param.data
                 try:
-                    param.copy_(input_param)
+                    with torch.no_grad():
```

Comment:
so if I'm reading this correctly, previously we didn't track autograd history through Parameters, but we did through Tensors?  And now we aren't tracking autograd history through either?  Why?

Reply:
`param` here is the result of `v.data` above. So changes to it are never tracked.
The new implementation makes that clearer that we never track history and just change the content in a non-differentiable manner.